crm_compassion/models/weekly_demand.py

```diff
@@ -147,13 +156,25 @@ def _default_cancellation(self):
         ])
         return float(cancellations) / 52.0
 
+    @api.model
+    def _default_sub_sponsorship(self):
+        """ Compute average of SUB since one year. """
+        start_date = datetime.today() - timedelta(weeks=52)
+        sub_sponsorship = self.env['recurring.contract'].search_count([
+            ('origin_id.type', '=', 'sub'),
+            ('start_date', '>=', fields.Date.to_string(start_date))
```

Comment:
@ecino I just remembered we have to come up with a suitable interval between the end date and the start date to consider it a sub sponsorship contract right ?